internal/engine/configs/actions.go

```diff
@@ -24,6 +24,11 @@ type ConfigsReloadedAction struct {
 	Warnings   []string
 	Features   map[string]bool
 	TeamName   string
+	Secrets    model.SecretSet
+
+	// The length of the global log when Tiltfile execution started.
+	// Useful for knowing how far back in time we have to scrub secrets.
```

Comment:
Is this a correctness issue? performance?

Tilt currently shows logs that predate Tilt execution, which seems contradictory to this.

I think that behavior is a bug, though.

Reply:
performance